frontend/src/pages/introduction.js

```diff
@@ -1,13 +1,26 @@
 import React from "react";
 import "bootstrap/dist/css/bootstrap.min.css";
-import { withStyles, Typography, Box, Grid, Button } from "@material-ui/core";
+import { withStyles, Typography, Box, Grid, Button, makeStyles } from "@material-ui/core";
 
 const TextTypography = withStyles({
   root: {
-    color: "#373a3c"
+    color: "#373a3c",
+    whiteSpace: 'pre-line'
   }
 })(Typography);
 
+const useStyles = makeStyles({
+  textBox:{
+    overflowY: 'auto',
+    maxHeight: window.innerHeight * 0.6,
+    marginTop: '30px', borderRadius: '5px',
```

Comment:
if you add (theme) => to your make styles definition, you can use theme.spacing(some integer) rather than hard coding px. Or use css rem. You just generally want to avoid hard coding px count as much as possible to ensure responsiveness across all screen sizes

Reply:
Done